test/integration/element_tracing.cc

```diff
@@ -283,6 +273,15 @@ TEST(ElementTracing, includes)
   EXPECT_EQ(14, overrideModelPluginElem->LineNumber().value());
   EXPECT_EQ(overrideModelPluginXmlPath, overrideModelPluginElem->XmlPath());
 
+#ifdef _WIN32
+  // There is a / in the last argument of TestFile here, as sdf::findFile does
+  // not sanitize the file paths provided, which in this case is
+  // 'test_model/model.sdf', therefore Windows machines will assign the path
```

Comment:
Can you reference the github issue here?

Reply:
https://github.com/osrf/sdformat/pull/578/commits/38b404299384d384f4ad17881fe4ae7d8169c9ad